_1327/documents/templates/editor.html

```diff
@@ -8,11 +8,15 @@
 		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "Please provide the missing information" %}</div>
 	{% endif %}
 
-    {% if form.autosave %}
-		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "This document was autosaved on" %} {{ form.autosave_date }}{% trans ", you can restore unsaved changes:" %} <a href="{{ edit_url }}?restore" class="btn-sm btn-default" role="button">{% trans "Click here" %}</a></div>
+	{% if form.autosave %}
+		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "This document was autosaved on" %} {{ form.autosave_date }}{% trans ", you can restore unsaved changes:" %} <a href="{% url 'information_pages:edit' document.url_title %}?restore" class="btn-sm btn-default" role="button">{% trans "Click here" %}</a></div>
```

Comment:
Don't use information page directly! The ``document`` app should not know about derived apps.